lib/swaggerValidator/trafficValidator.ts

```diff
@@ -111,6 +177,44 @@ export class TrafficValidator {
         ],
       });
     }
+    this.operationSpecMapper.forEach((value: string[], key: string) => {
+      if (this.trafficOperation.get(key) === undefined) {
+        this.coverageResult.set(key, 0);
+        this.coverageData.push({
+          spec: key,
+          coveredOperaions: 0,
+          totalOperations: value.length,
+          coverageRate: 0,
+        });
+      } else {
+        if (value !== undefined && value.length !== 0) {
+          this.coverageResult.set(key, this.trafficOperation.get(key)!.length / value.length);
```

Comment:
How about use a variable to keep `this.trafficOperation.get(key)` which is called triple times?

Reply:
What's the change or comment for this issue?